doc/maintaining/installing/install-from-source.rst

```diff
@@ -24,10 +24,26 @@ work on CKAN.
 1. Install the required packages
 --------------------------------
 
-If you're using a Debian-based operating system (such as Ubuntu) install the
-required packages with this command::
+CKAN is a Python application that requires three main services: PostgreSQL, Solr and Redis.
 
-    sudo apt-get install python3-dev postgresql libpq-dev python3-pip python3-venv git-core solr-tomcat openjdk-8-jdk redis-server
+Python required packages::
+
+    sudo apt-get install python3-dev libpq-dev python3-pip python3-venv git-core
+
+The 3 main services (PostgreSQL, Solr and Redis) can be installed locally or dockerized. If
+you want to run them locally you will need to install the following dependencies.
+
+PostgreSQL requirements::
+
+    sudo apt-get install postgresql
+
+Solr requirements::
+
+    sudo apt-get install solr-tomcat openjdk-8-jdk
```

Comment:
I think we should encourage people to use the Dockerized Solr version whenever possible, so let's remove these instructions for postgres and solr, as people will find them in their dedicated pages (actually check the postgres one). redis-server can be added back to the packages above

Reply:
Done @amercader !